resources/views/common/modals/create_reference.html.twig

```diff
@@ -193,4 +194,4 @@
 
     </div>
 
-</div>
+<!--</div>-->
```

Comment:
Curious why this closing tag is commented?

Reply:
Removed it, was a stray end tag on reference pages.

Reply:
Are we sure this particular close tag is the culprit? This is a pretty small file with paired open/close tags... I have a suspicion the larger more complicated template that includes this one is the problem.

Reply:
I've restored it. Can't reproduce the stray tag error now.